core/src/main/java/org/mskcc/cbio/portal/dao/GeneticAlterationUtil.java

```diff
@@ -47,9 +51,19 @@
  *
  * @author Ethan Cerami.
  */
+@Component
 public class GeneticAlterationUtil {
     private static final String NAN = "NaN";
 
+    private static MutationRepository mutationRepository;
+    private static MutationModelConverter mutationModelConverter;
+
+    @Autowired
```

Comment:
How will this work when classes not managed by Spring rely on this class (like GetAlterationDataJSON, GetProfileDataJSON, GetGeneticProfileJSON, GetProfileData)?